frontstage/controllers/collection_instrument_controller.py

```diff
@@ -86,3 +86,14 @@ def upload_collection_instrument(upload_file, case_id, party_id):
         raise ApiError(logger, response)
 
     logger.info('Successfully uploaded collection instrument', case_id=case_id, party_id=party_id)
+
+
+def check_collection_instrument_size(upload_file, party_id, case_id):
```

Comment:
This function name should probably be `is_collection_exercise_too_small`, because that's what it's doing.  It also makes it instantly understandable by just looking at the function name.

Also, why are we passing in the party_id and case_id if all we're doing is logging what they are.  Agreed we should be adding the context to the logs where possible, but I don't think passing the values to the function exclusively for logging is correct

Also add a docstring!